lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -38,18 +38,25 @@ function focusInInput(event: Event) {
   return /input|textarea/i.test(target.tagName) || target.getAttribute('contenteditable') !== null;
 }
 
-function createController() {
+function createController(): AbortController {
   if (AbortController) return new AbortController();
   // Polyfill for IE11
   return {
     signal: { aborted: false },
     abort() {
       this.signal.aborted = true;
     },
-  };
+  } as AbortController;
```

Comment:
if you're casting here, do you need to define the type in line 41 as well?

Reply:
Seems so, yes. Don't know why TS doesn't figure it out on its own.